refactor/imports.go

```diff
@@ -39,6 +39,11 @@ func deleteUnusedImports(s *Snapshot, p *Package, text []byte) []byte {
 	match := func(name, pkg string) bool {
 		if name == "" {
 			p1 := s.pkgGraph.byPath(pkg)
+			if p1 == nil {
+				if val, ok := p.ImportMap[pkg]; ok {
```

Comment:
https://github.com/rsc/rf/blob/main/refactor/snap.go#L837-L843 checks ImportMap before `s.pkgGraph.byPath`. Here we do the opposite.

I don't think the order matter, just food for thought if you know of cases where order would matter.